### PR TITLE
[0.13.x] Bump version to 0.13.3.0 and add release notes

### PR DESCRIPTION
**Pull request by codablock**

### CODE REVIEW

**Comment by UdjinM6** [comment]
Current `doc/release-notes.md` should be copied to `doc/release-notes/dash/release-notes-0.13.2.md`.

**Comment by UdjinM6** [comment]
Should also update `Credits` and `Older releases`

**Comment by codablock** [comment]
@UdjinM6 done

File: doc/release-notes.md [hunk 1]
```diff
@@ -37,95 +37,62 @@ Downgrade warning
 Downgrading to a version smaller than 0.13 is not supported anymore as DIP2/DIP3 has activated
 on mainnet and testnet.
 
-### Downgrade to 0.13.0.0 or 0.13.1.0
+### Downgrade to a version < 0.13.3.0
 
-Downgrading to 0.13.0.0 is fully supported but is not recommended unless you have some serious issues with 0.13.2.0.
+Downgrading to 0.13.3.0 is fully supported but is not recommended unless you have some serious issues with 0.13.3.0.
```
Comment (UdjinM6) [inline doc/release-notes.md]:
```suggestion
Downgrading to previous 0.13 releases is fully supported but is not recommended unless you have some serious issues with 0.13.3.0.
```

Comment (UdjinM6) [reply doc/release-notes.md]: or smth like that

**Pull request merged**